descriptors/ModuleDescriptor-template.json

```diff
@@ -58,7 +58,11 @@
         {
           "methods": ["POST"],
           "pathPattern": "/invoice/invoice-lines",
-          "permissionsRequired": ["invoice.invoice-lines.item.post"]
+          "permissionsRequired": [
+            "invoice.invoice-lines.item.post",
+            "invoice-storage.invoice-line-number.get",
```

Comment:
the invoice-storage permissions must be set under "modulePermissions"

Reply:
Done

Reply:
Also I think the invoice lines permission is missing- please add "invoice-storage.invoice-lines.item.post"

Reply:
Wait, should we really be requiring invoice-storage permissions for an invoice endpoint?  I don't think we should

Reply:
Done

Reply:
Also please add the invoice-lines interface dependency, Sorry Just noticed it

Reply:
Got it. Done